### DISCUSSION TITLE
RustDeskCustom Client => Every Remote Control setting disabled?

### DISCUSSION THREAD

**Discussion by nicedevil007** [rustdesk/rustdesk-server-pro#275] [Q&A]
Hey guys,

I'm using the new custom client and I can't remote control anything on the remote device after putting in my password.
I'm not sure if there is any setting on the custom client that disables all the options.
The remote PC's user is still able to allow all functions while clicking on all the possible buttons (remote control, remote restart, copy/paste etc.)

I want some parts of this to be default.

# Strategy
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/17103076/f3de205b-d1f2-4755-9f85-7c3d175f3dbd)

# Custom client is setup
## GUI Settings
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/17103076/063781c7-1dee-49a6-adce-dbf07004a71f)

### Override settings:
```yaml
access-mode=custom
enable-keyboard=Y
enable-clipboard=Y
enable-file-transfer=Y
enable-audio=N
enable-tunnel=N
enable-remote-restart=N
enable-record-session=N
allow-remote-config-modification=N
enable-lan-discovery=N
direct-server=N
allow-auto-disconnect=N
auto-disconnect-timeout=60
allow-only-conn-window-open=N
theme=dark
lang=de
allow-auto-record-incoming=N
allow-always-software-render=N
allow-linux-headless=N
enable-hwcodec=Y
show-monitors-toolbar=Y
show-quality-monitor=N
disable-audio=Y
enable-file-copy-paste=Y
disable-clipboard=N
lock-after-session-end=N
privacy-mode=N
whitelist=,
preset-address-book-name=************
preset-address-book-tag=workstations
```

### Default settings:
```yaml
enable-abr=Y
allow-remove-wallpaper=N
enable-open-new-connections-in-tabs=Y
peer-card-ui-type=2
peer-sorting=Remote Host
view-only=N
collapse-toolbar=Y
show-remote-cursor=Y
follow-remote-cursor=Y
follow-remote-window=Y
zoom-cursor=Y
view-style=adaptive
enable-block-input=N
```

**Comment by rustdesk** [comment]
@fufesou follow it up

**Comment by fufesou** [comment]
@nicedevil007 Do you means you want to modify the secure settings on the control side? 

The following option in your override settings is `N`, you can try `Y`.

```
allow-remote-config-modification=Y
```

**Reply by nicedevil007** [reply]
I guess this is not what I mean, I understand this option as it will allow the connecting user to be able to modify settings inside the rustdesk of the host, isn't it?

I just want the default options turned on at the start of a session.

default for me is, this:

```yaml
enable-keyboard=Y
enable-clipboard=Y
enable-file-transfer=Y
```

**Reply by fufesou** [reply]
Sorry, I cann't understand clearly.

The options are turned on, and the remote user can turn on the other unchecked options.

<img width="433" alt="1719144377882" src="https://github.com/rustdesk/rustdesk-server-pro/assets/13586388/fbdfbaac-f259-4ea0-bfbf-0916f9e96568">

You can also put the permission settings in the "Default settings" if you want the let them editable for the remote user

```
access-mode=custom
enable-keyboard=Y
enable-clipboard=Y
enable-file-transfer=Y
enable-audio=N
enable-tunnel=N
enable-remote-restart=N
enable-record-session=N
allow-remote-config-modification=N
```

<img width="433" alt="1719144377882" src="https://github.com/rustdesk/rustdesk-server-pro/assets/13586388/c3920e16-d390-445c-ae80-ac15a7945db7">

**Reply by nicedevil007** [reply]
Ah thank you for the screenshot :) This will help to make you understand my issue:

![image](https://github.com/rustdesk/rustdesk-server-pro/assets/17103076/bf66cc47-8a70-4c0d-b08a-4198c0fe9d42)
on your screen on "Berechtigungen" the first 2 icons and the 4th one is already "enabled".

If I connect to a PC from my workstation, everything is greyed out and I can't do anything. I want a default setting that those 3 icons are turned blue/enabled on connection without the need to enable those by the support asking user.

---------------

Da du nen deutsches RustDesk hast, gehe ich mal von aus, dass du dieser Sprache mächtig bist :)
Vielleicht ist mein englisch zu doof für das zum Beschreiben.

Also man kann doch dafür sorgen, dass alle Berechtigungen per Default an oder aus sind und ich habe meines erachtens, Keyboard/Maus, Clipboard und Filetransfer enabled, allerdings wird alles ausgegraut bei mir angezeigt nach dem connecten zu einer RemoteWorkstation.

**Reply by fufesou** [reply]
> If I connect to a PC from my workstation, everything is greyed out and I can't do anything.

Can you please give a screenshot of the permissions on the controlled side?

<img width="324" alt="1719146413433" src="https://github.com/rustdesk/rustdesk-server-pro/assets/13586388/6571ee03-6f71-43a2-98be-5d1be138cfe1">

My RustDesk is in German because I used the same settings as you.

**Reply by nicedevil007** [reply]
> > If I connect to a PC from my workstation, everything is greyed out and I can't do anything.
> 
> Can you please give a screenshot of the permissions on the controlled side?
> 
> <img alt="1719146413433" width="324" src="https://private-user-images.githubusercontent.com/13586388/342065153-6571ee03-6f71-43a2-98be-5d1be138cfe1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkxNDcwODEsIm5iZiI6MTcxOTE0Njc4MSwicGF0aCI6Ii8xMzU4NjM4OC8zNDIwNjUxNTMtNjU3MWVlMDMtNmY3MS00M2EyLTk4YmUtNWQxYmUxMzhjZmUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIzVDEyNDYyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxYjVhZTE4NDM4YTliZmZjODBlNjk0Mzg4NTk0MTYyYmY4M2U2Njk1MGU1OTBmZWQ1OWVhYThiNTVjMjA0Y2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Y_cT8LQC1TCnaMg1gIHppX24nEl23yRkZoAthSwBPKQ">
> My RustDesk is in German because I used the same settings as you.

I can't because settings for the endusers are disabled.

**Reply by nicedevil007** [reply]
Ok wait, I tryed to connect with the opposit direction, so from the "old" controlled side to my PC... then it works, but I did a fresh install of my custom client here... I guess something is broken then with the installation routine, so it is not working with the right installer :(

**Reply by fufesou** [reply]
Maybe the controlled side is not the custom client build or it is not installed correctly.

**Reply by nicedevil007** [reply]
> Maybe the controlled side is not the custom client build or it is not installed correctly.

this is the case, went through my whole MECM install routine over PSAppDeployToolkit and found the issue... was an old install that was used there from the cache.
